packages/editor-ui/src/mixins/workflowRun.ts

```diff
@@ -175,43 +175,13 @@ export const workflowRun = defineComponent({
 
 				const runData = this.workflowsStore.getWorkflowRunData;
 
-				let newRunData: IRunData | undefined;
+				const newRunData = Object.entries(runData || {}).reduce((acc, [key, value]) => {
+					acc[key] = value.slice(0, 1);
+					return acc;
+				}, {} as IRunData);
```

Comment:
can you please add some tests? definitely an e2e for this bug. And maybe some unit tests here.
otherwise we are bound to repeat this bug